### PR TITLE
chore: bump versions

### PR DESCRIPTION
**Pull request by utkuozdemir**
Bump playwright base image and go versions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM mcr.microsoft.com/playwright:v1.35.1-focal AS container
+FROM mcr.microsoft.com/playwright:v1.40.0-jammy AS container
```
Comment (frezbo) [inline Dockerfile:1]: We can use `v1.40.1-jammy`

**Review by frezbo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (utkuozdemir) [reply Dockerfile:1]: Ah good one, even in their docs they have 1.40.0 all over the place. Where did you find it?

Comment (frezbo) [reply Dockerfile]: `crane ls mcr.microsoft.com/playwright`

**Pull request merged**